R/dockerfile.R

```diff
@@ -1,24 +1,16 @@
-# Copyright 2016 Daniel Nüst
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
+# Copyright 2016 Opening Reproducible Research (http://o2r.info)
 
 #' dockerfile
 #'
 #' Create a Dockerfile from the
 #'
 #' @param from the source of the information to construct the Dockerfile
-#' @param to path and file name to save the Dockerfile to
+#' @param env the environment that should be included in the image
```

Comment:
Well. This happened when I tried to make the vignette run. There was a call as follows:

`dockerfile(sessionInfo(), ls())`

Obviously, my attempt to give the second parameter an appropriate name and description was a little helpless. I thought about R environments, e.g. the global environment, but actually what this parameter takes is a list of objects to be included in the session.